doc/man7/provider-signature.pod

```diff
@@ -234,6 +300,34 @@ is I<tbslen> bytes long.
 The signature is pointed to by the I<sig> parameter which is I<siglen> bytes
 long.
 
+=head2 Message Verify Functions
+
+These functions are suitable to algorithms that implement algorithms that
+accumulates a full message and signs the result of that accumulation, such as
+RSA-SHA256.
+
+OSSL_FUNC_signature_verify_message_init() initialises a context for signing a
+message given a provider side signature context in the I<ctx> parameter, and a
+pointer to a provider key object in the I<provkey> parameter.
+The I<params>, if not NULL, should be set on the context in a manner similar to
+using OSSL_FUNC_signature_set_ctx_params().
+The key object should have been previously generated, loaded or imported into
+the provider using the key management (OSSL_OP_KEYMGMT) operation (see
+provider-keymgmt(7)>.
+
+OSSL_FUNC_signature_verify_message_update() gathers the data pointed at by
+I<in>, which is I<inlen> bytes long.
+
+OSSL_FUNC_signature_verify_message_final() performs the actual signing on the
+data that was gathered with OSSL_FUNC_signature_verify_message_update().  The
+signature itself must have been passed through the "signature"
+(B<OSSL_SIGNATURE_PARAM_SIGNATURE>) L<Signature parameter|/Signature parameters>
+before this function is called.
+
+OSSL_FUNC_signature_verify() can be used for one-shot signature calls.  In that
+case, I<tbs> is expected to be the whole message to be signed, I<tbslen> bytes
```

Comment:
Do we normally use `tbs` for verify operations?

Reply:
Referring back to 3.3.2:

https://github.com/openssl/openssl/blob/fb7fab9fa6f4869eaa8fbb97e0d593159f03ffe4/doc/man3/EVP_PKEY_verify.pod#L14-L16
https://github.com/openssl/openssl/blob/fb7fab9fa6f4869eaa8fbb97e0d593159f03ffe4/doc/man7/provider-signature.pod#L34-L35